api/libexec/slurm/monitor.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
```

Comment:
Is this file still needed?  Is it not part of the CLI now?

Reply:
Same question for all the files in the `libexec` directory.

Reply:
e0821664dbe19fed298c0c37846652b20d0e34fa

Reply:
e0821664dbe19fed298c0c37846652b20d0e34fa